app/controllers/Application.java

```diff
@@ -1797,44 +1802,43 @@ public static Result getTuningParameter(String jobId) {
+      logger.info("Respective JobSuggestedParamSet: " + jobSuggestedParamSet.id);
       if (userSuggestedParamSet == null) {
-        logger.info("No user suggested param set for jobDefinitionId:" + jobDefinitionId);
+        logger.info("No user suggested param set for jobDefinitionId: "  + jobDefinitionId);
       } else {
         logger.info("User suggested param set: " + userSuggestedParamSet.id);
       }
       logger.info("Tuning Job Definition Id: " + tuningJobDefinition.job.id);
       Boolean autoApply = tuningJobDefinition.autoApply;
       TuningAlgorithm tuningAlgorithm = jobSuggestedParamSet.tuningAlgorithm;
       List<TuningParameter> parametersList = getTuningParametersListForJob(tuningAlgorithm.id);
-      logger.info("Tuning Algorithm id: " + tuningAlgorithm.id);
       JsonArray tuningParameters = getTuningParameterDetails(parametersList, jobSuggestedParamSet, userSuggestedParamSet);
       String currentTuningAlgorithm = getCurrentTuningAlgorithmName(tuningAlgorithm);
       JsonArray tuningAlgorithmList = new JsonArray();
-      tuneIn.addProperty("id", jobId);
-      tuneIn.addProperty("jobDefinitionId", jobDefinitionId);
+      tuneIn.addProperty(ID, jobId);
+      tuneIn.addProperty(JOB_DEFINTITION_ID, jobDefinitionId);
       //Two tuning types {HBT, OBT}
       JsonObject hbtAlgo = new JsonObject();
-      hbtAlgo.addProperty("name", "HBT");
+      hbtAlgo.addProperty(NAME, TuningType.HBT.name());
       JsonObject obtAlgo = new JsonObject();
-      obtAlgo.addProperty("name", "OBT");
+      obtAlgo.addProperty(NAME, TuningType.OBT.name());
       tuningAlgorithmList.add(hbtAlgo);
       tuningAlgorithmList.add(obtAlgo);
-      tuneIn.addProperty("jobSuggestedParamSetId", jobSuggestedParamSet.id);
-      tuneIn.addProperty("tuningAlgorithmId", tuningAlgorithm.id);
-      tuneIn.addProperty("autoApply", autoApply);
-      tuneIn.addProperty("tuningAlgorithm", currentTuningAlgorithm);
-      tuneIn.addProperty("reasonForTuningDisable", reasonForDisablingTuning(tuningJobDefinition));
-      tuneIn.add("tuningAlgorithmList", tuningAlgorithmList);
-      tuneIn.addProperty("iterationCount",tuningJobDefinition.numberOfIterations);
-      tuneIn.add("tuningParameters", tuningParameters);
-      parent.add("tunein", tuneIn);
-      logger.debug("tuneIn Json: " + tuneIn);
+      tuneIn.addProperty(JOB_SUGGESTED_PARAM_SET_ID, jobSuggestedParamSet.id);
+      tuneIn.addProperty(TUNING_ALGORITHM_ID, tuningAlgorithm.id);
+      tuneIn.addProperty(AUTO_APPLY, autoApply);
+      tuneIn.addProperty(TUNING_ALGORITH, currentTuningAlgorithm);
+      tuneIn.addProperty(REASON_TO_DISABLE_TUNING, reasonForDisablingTuning(tuningJobDefinition));
+      tuneIn.add(TUNING_ALGORITHM_LIST, tuningAlgorithmList);
+      tuneIn.addProperty(ITERATION_COUNT,tuningJobDefinition.numberOfIterations);
+      tuneIn.add(TUNING_PARAMETERS, tuningParameters);
+      parent.add(TUNEIN, tuneIn);
+      logger.debug("TuneIn Json: " + tuneIn);
```

Comment:
check for isDebugEnabled before doing logger.debug

Reply:
Sure